ci-operator/step-registry/workers-rhel/vsphere-provision/workers-rhel-vsphere-provision-commands.sh

```diff
@@ -47,27 +47,43 @@ for count in $(seq 1 ${RHEL_WORKER_COUNT}); do
     exit 1
   fi
 
-  echo ${rhel_node_ip} >> /tmp/rhel_nodes_ip
+  echo "${rhel_node_ip}" >> /tmp/rhel_nodes_ip
+  echo "${infra_id}-rhel-${count},${rhel_node_ip}" >> /tmp/rhel_nodes_info
 done
 
+lb_group=""
+lb_vars=""
+if test -n "$(govc ls ${vcenter_folder} | grep "lb-0")"; then
+  lb_ssh_user=$(</var/run/vault/vsphere/vsphere_lb_ssh_user)
```

Comment:
Because the lb is launched by UPI script, its ssh user is only `core`, so I am thinking if we can hard code it to be `core`, so that do not have to introduce a new vault value.

Reply:
okay, will change to hardcode